src/components/DeferLoad/index.ts

```diff
@@ -114,13 +115,15 @@ class DeferLoad extends BridgeBase {
       // Rendering defer loaded tabs in case/ page context
       this.thePConn
         .getActionsApi()
+        // @ts-ignore
```

Comment:
Can you leave a comment as to why this ts-ignore is needed so we can pass this back to core team